sickbeard/webserveInit.py

```diff
@@ -201,4 +204,4 @@ def autoproxy(
     )
 
     cherrypy.server.start()
-    cherrypy.server.wait()
+    #cherrypy.server.wait()

```

Comment:
I found that the .wait() method doesn't take any real effect beyond delaying the time the service comes up to an available state.  Is there a defined reason to delay here?